src/shared/telemetry/defaultTelemetryService.ts

```diff
@@ -61,19 +62,7 @@ export class DefaultTelemetryService implements TelemetryService {
     }
 
     public async start(): Promise<void> {
-        this.record(
-            {
-                createTime: this.startTime,
-                data: [
-                    {
-                        MetricName: 'session_start',
-                        Value: 0,
-                        Unit: 'None'
-                    }
-                ]
-            },
-            this.awsContext
-        )
+        recordSessionStart()
```

Comment:
What does this do that `this.startTimer` doesn't?

Reply:
This does the record of `session_start`?

Reply:
Oh duh